services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -201,6 +204,14 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
     dispatch(change(FORM.VIEW_EDIT_REPORT, "documents", docs));
   };
 
+  const handleOpenMoreInformation = () => {
+    const newWindow = window.open(
```

Comment:
do we have to use window.open here or can the button be a link instead? I see an example of a Button link in PartyOrgBookForm.tsx (I searched codebase for _blank)

Reply:
Updated.